### ISSUE TITLE
Simple Shadows working?

### ISSUE THREAD

**Issue by geometry** [ellioman/Indirect-Rendering-With-Compute-Shaders#3]
Hi,
This is an exciting project. I'm trying it out to render detail meshes on other meshes (transferred from a terrain).
The 'Enable Simple Shadows' option seems to have no effect in the sample scene. 
I've tried disabling the scene view. 
I'm using Unity 2018.2.18f1.
Thanks.

**Comment by ellioman** [comment]
Heyhey,

That is weird. I'll try to take a look today/tonight and see if I accidentally created a bug there.
Will reply as soon as I know something.

Cheers,
Elvar

**Comment by ellioman** [comment]
Howdy, 
Sorry for the late reply, work plus Christmas stealing my time :)

I'm using Unity 2018.2.20f1 and toggling the simple shadows is working fine on my Mac. 
Are you using a PC or Mac? 

![screenflow](https://user-images.githubusercontent.com/225349/50820953-434c2d80-132e-11e9-938c-2b755737bf0f.gif)

**Comment by teleceratops** [comment]
Hi,
Likewise: apologies for the late response. I'm on a PC. I've tried it on a couple of different NVidia and an AMD card, all fairly high-end with up-to-date drivers. I guess Mac users don't have these kinds of problems whereas we're used to weird stuff on PCs.
![image](https://user-images.githubusercontent.com/5726132/51057124-93501c00-15dc-11e9-908e-b3c75232f3fd.png)
I'll dig about a bit to see if I can find the problem...but don't hold your breath!

Steve

**Comment by ellioman** [comment]
Interesting, There must be a mistake somewhere on my part. I willl download the project to my work computer (Windows 10 & NVidia card) tomorrow and see I can't reproduce it :)

**Comment by teleceratops** [comment]
I've just updated to 2018.3.1f1 and I have shadows!
![image](https://user-images.githubusercontent.com/5726132/51085055-e97fa500-172b-11e9-8c9e-b169149d931f.png)
I'll play around a little more until I'm happy. I'll retest your original distro without my fiddles and perhaps close this issue later today.
Hmmm.

**Comment by teleceratops** [comment]
![image](https://user-images.githubusercontent.com/5726132/51088519-98d27100-1758-11e9-9824-3f125032ba08.png)
My problem was quality settings and the Shadow Distance. I only realised when I saw that the occluders also had no shadow. Many thanks for your help. I look forward to seeing what's possible.
![image](https://user-images.githubusercontent.com/5726132/51088551-f8308100-1758-11e9-8da0-176cf5d7f260.png)
